prep-css-flexbox/flex-wrap.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Flexbox - flex-wrap</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="starwars.css">
+  <style>
+    /* your styles here 👇 */
+    div {
+      flex-basis: calc(100% / 6);
+    }
```

Comment:
Rather than apply `flex-basis` to all `div` elements, it would be more precise to apply it to all `.card-wrappers` instead.

Reply:
![image](https://user-images.githubusercontent.com/96964291/149857401-244dabc5-9d36-49a6-9ad2-4c158e9f90a8.png)
My bad! I have corrected it as directed, thank you for the feedback!